### ISSUE TITLE
Add dataset migration CLI command

### ISSUE THREAD

**Issue by TheAndrewJackson** [ethyca/fides#1764]
❗ Depends on #1763 ❗ 

Now that the `dataset` column on `datasetconfig` is no longer being used users need a way to merge all of the datasets they have stored in `datasetconfig` into `ctl_datasets`.

A high level idea of what this flow would like is:
- A new `dataset-migration` command is added to the cli
- When a user runs that command the `datasetconfig` table is scanned and every row that contains config data stored on the `dataset` column is converted into a yaml file and merged into the corresponding `ctl_datasets` config yaml file locally on the users machine based on the `fides_key`. 
- The user would then look at the git diffs and manually merge them together. 
- Once everything looks good they can check the merged files into source control and push the changes up to the fides server

This flow would also handle renaming the `fidesops_meta` to `fides_meta`. It would also need a way to mark the datasets that have been converted as such so they aren't scanned and merged again. 

This command won't delete any data. It's just simplifying the merge process. The legacy data will stored be stored for now but not accessible through the api.

**Comment by TheAndrewJackson** [comment]
@ThomasLaPiana Does this match what you had in mind when this was discussed in the design doc review?

**Comment by ThomasLaPiana** [comment]
@TheAndrewJackson yep!

Still some tricky implementation details here, but those can wait for the PR :)

**Comment by seanpreston** [comment]
Whether we action this ticket will depend on the following question:

Are there live Fides deployments where the same dataset exists between both tables, such that they might conflict?

**Comment by pattisdr** [comment]
https://github.com/ethyca/fides/pull/2046 solves this in a simpler way by adding an automated data migration that copies over all DatasetConfig.datasets into the ctl_dataset table, creating new records for each.  If there is a fides_key conflict, the migration will error and the user should manually resolve.
